### PR TITLE
to lerna

### PR DESCRIPTION
**Pull request by viliusl**
@ronami this is task to get rid of octopus (but sadly we need a bit more that lerna provides - lerna-script).

I dropped things that were annoying (dep management etc.), but:
 - bootstrap via lerna;
 - run tests via lerna-script. mostly because we need to work-around where yoshi and travis do not get along;
 - sync `yarn.lock` and `.nvmrc` via husky and prepush hooks.

Also ran a sync task and normalized homepage and repo links in package.json's so that `npm docs yoshi` would work correctly.

### CODE REVIEW

**Comment by ronami** [comment]
@viliusl it looks great, is it a wip or can we merge?

Also, would it be possible to run a different reporter for mocha (progress for instance) so we don't need to work around travis?

**Comment by viliusl** [comment]
@ronami - it was done, but moved it to WIP again - liked your idea regarding custom reporter for travis - will try it out:)

**Comment by ronami** [comment]
@viliusl actually I don't think a custom reporter would help because we need to silence all output except for some ticks to keep travis alive. I don't think reporters can actually silent the output coming from tests as it's all just the same process.

**Comment by viliusl** [comment]
@ronami I'm done here. Left only minimal set-up for build to work/pass in ci + local bootstrap.

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "lerna": "2.0.0",
```
Comment (ronami) [inline lerna.json]: Why not use latest (2.5.0?)

**Pull request merged**